src/EnergyPlus/StandardRatings.cc

```diff
@@ -1852,18 +1860,22 @@ namespace StandardRatings {
         static Real64 CoolingElecPowerMax(0.0);          // outdoor unit electric power input at Max speed, [W]
         static Real64 PartLoadRatio(0.0);                // compressor cycling ratio between successive speeds, [-]
         static Real64 PartLoadFraction(0.0);             // part-load fraction that account for the cyclic degradation, [-]
+        static Real64 PartLoadFractionDefault(0.0);      // part-load fraction that account for the cyclic degradation from AHRI 201/240 default PLF curve and C_D value, [-]
         static Real64 NetCoolingCapWeighted(0.0);        // net tot cooling cap weighted by the fraction of the binned cooling hours [W]
-        static Real64 TotCoolingElecPowerWeighted(0.0);  // net total cooling electric power input weighted by the fraction of the
+        static Real64 TotCoolingElecPowerWeighted(0.0);  // net total cooling electric power input weighted by the fraction of the temperature bins
+        static Real64 TotCoolingElecPowerWeightedDefault(0.0);  // net total cooling electric power input weighted by the fraction of the temperature bins from AHRI 201/240 default PLF curve and C_D value,
         // binned cooling hours
         static Real64 BuildingCoolingLoad(0.0);    // Building space cooling load corresponding to an outdoor bin temperature [W]
         static Real64 NetTotCoolCapBinned(0.0);    // Net tot cooling cap corresponding to an outdoor bin temperature [W]
         static Real64 TotCoolElecPowerBinned(0.0); // Total cooling electric power corresponding to an outdoor bin temperature [W]
+        static Real64 TotCoolElecPowerBinnedDefault(0.0); // Total cooling electric power corresponding to an outdoor bin temperature from AHRI 201/240 default PLF curve and C_D value, [W]
         static Real64 LoadFactor(0.0);             // "on" time for last stage at the desired reduced capacity, (dimensionless)
         int BinNum;                                // bin number counter
         int spnum;                                 // compressor speed number
 
+        NetCoolingCapWeighted = 0.0;
         TotCoolingElecPowerWeighted = 0.0;
-        TotCoolingElecPowerWeighted = 0.0;
+        TotCoolingElecPowerWeightedDefault = 0.0;
```

Comment:
TotCoolingElecPowerWeighted was duplacte. Initialized NetCoolingCapWeighted local variable. Otherwise, the later would have incorrect results when you have multiple call SEER calc function for multispeed DX coils. Found out during unit test.